site2/docs/admin-api-brokers.md

```diff
@@ -4,16 +4,26 @@ title: Managing Brokers
 sidebar_label: Brokers
 ---
 
+> **Important**
```

Comment:
Do we need to add the same note everywhere in the docs? If it's general note that users need to pay attention to, it's better to highlight it in a central place, e.g.: overview.

We do not need to replace the same content again and again in all docs.

Reply:
Are you suggesting putting it in a separate "overview" documentation page?

That doesn't really solve the problem here, which is to make sure someone who lands on this page is aware that it's not a complete description of all endpoints.

What if we just eliminate this page altogether and use the generated pages instead? I don't see why we need a separate "frequently used operations" page and another complete reference.

Reply:
I'm not suggesting adding a separate "overview" doc page, my concern is: if we add the same content so much times, we need to consider adding it once in somewhere (that users visit most frequently, for example the "overview" of this whole chapter), and reference it when needed in other places. It's easy to maintain and keep consistent for a long run. We don't need to refine the same content many times manually in the future.

Reply:
@Jennifer88huang for this concern, I've replied before: https://github.com/apache/pulsar/pull/11146#issuecomment-870483161

I think this is a temporary workaround (to let users know the info quickly since many users have this issue and writers need to reply one by one), we may consider adding a floating window on the website, need more research later.